test/guest/network-cockpit.xml

```diff
@@ -1,4 +1,4 @@
-<network>
+<network ipv6='yes'>
```

Comment:
No and that's why I want to get this in place now. And then at some point we can actually make use of this in the future.